scipy/integrate/_ivp/tests/test_ivp.py

```diff
@@ -665,7 +676,7 @@ def test_ConstantDenseOutput():
 
 def test_classes():
     y0 = [1 / 3, 2 / 9]
-    for cls in [RK23, RK45, Radau, BDF, LSODA]:
+    for cls in [RK23, RK45, DOP853, Radau, BDF, LSODA]:
```

Comment:
You parametrize the new test file nicely below, but actually not parametrizing here might the right choice for now--helps keep the diff clean of extra refactoring for a PR that is already quite large.

Reply:
I guess this more logically fits into another pull request indeed.